.circleci/config.yml

```diff
@@ -1,9 +1,11 @@
-version: 2
+version: 1
+job_setup: 
```

Comment:
we can change this with something named like `build` that also does the `checkout` and `npm install` steps, as they seem to be repeated on each job